src/gcp_scanner/scanner.py

```diff
@@ -129,166 +130,104 @@ def save_results(res_data: Dict, res_path: str, is_light: bool):
     outfile.write(sa_results_data)
 
 
-def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
-               out_dir: str,
-               scan_config: Dict,
-               light_scan: bool,
-               target_project: Optional[str] = None,
-               force_projects: Optional[str] = None):
-  """The main loop function to crawl GCP resources.
-
-  Args:
-    initial_sa_tuples: [(sa_name, sa_object, chain_so_far)]
-    out_dir: directory to save results
-    target_project: project name to scan
-    force_projects: a list of projects to force scan
-  """
-
-  # Generate current timestamp to append to the filename
-  scan_time_suffix = datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
-
-  context = SpiderContext(initial_sa_tuples)
-  # Main loop
-  processed_sas = set()
-  while not context.service_account_queue.empty():
-    # Get a new candidate service account / token
-    sa_name, credentials, chain_so_far = context.service_account_queue.get()
-    if sa_name in processed_sas:
-      continue
-
-    # Don't process this service account again
-    processed_sas.add(sa_name)
-    logging.info('>> current service account: %s', sa_name)
-    sa_results = infinite_defaultdict()
-    # Log the chain we used to get here (even if we have no privs)
-    sa_results['service_account_chain'] = chain_so_far
-    sa_results['current_service_account'] = sa_name
-    # Add token scopes in the result
-    sa_results['token_scopes'] = credentials.scopes
-
-    project_list = CrawlerFactory.create_crawler(
-      'project_list',
-    ).crawl(
-      ClientFactory.get_client('cloudresourcemanager').get_service(credentials),
+def get_resources(project):
```

Comment:
Please follow our function definition guideline. We usually have parameter types and function description